build.gradle

```diff
@@ -0,0 +1,31 @@
+plugins {
+    id 'org.springframework.boot' version '2.1.3.RELEASE'
+    id 'java'
+}
+
+apply plugin: 'io.spring.dependency-management'
+
+group = 'pl.edu.agh'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+    compileOnly {
+        extendsFrom annotationProcessor
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+    implementation 'org.springframework.boot:spring-boot-starter-data-rest'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    compileOnly 'org.projectlombok:lombok'
+    runtimeOnly 'com.h2database:h2'
+    annotationProcessor 'org.projectlombok:lombok'
+    testImplementation 'org.springframework.boot:spring-boot-starter-test'
+    testCompile group: 'org.spockframework', name: 'spock-core', version: '1.3-groovy-2.5'
```

Comment:
Hm, tutaj sobie wrzucamy Spocka w zależności, a przykładowy test mamy tylko JUnitowy - nie wiem, czy mi się to podoba. Może po prostu dodamy, jak będzie nam potrzebny? Zresztą pisanie testów w dwóch różnych językach nie do końca do mnie przemawia

Jakby miał zostać, to proponuję ujednolicenie formatu na:
`testCompile 'org.spockframework:spock-core:1.3-groovy-2.5`

Do tego z doświadczenia do Spocka warto dorzucić:
```
    // allows mocking of classes (in addition to interfaces)
    testRuntime "net.bytebuddy:byte-buddy:1.9.10"
    // allows mocking of classes without default constructor (with CGLIB)
    testRuntime "org.objenesis:objenesis:3.0.1"
```
Do tego, jakby miał zostać, to chyba jeszcze byśmy potrzebowali `id 'groovy'` w plugins, ale ja bym go wywaliła, skoro w szkielecie projektu jest tylko Java